doc/changes/changes_2.8.0.md

```diff
@@ -0,0 +1,73 @@
+# Cloud Storage Extension 2.8.0, released 2024-05-17
+
+Code name: Simplify GCS Configuration
+
+## Summary
+
+This release allows configuring Google Cloud Storage (GCS) via a `CONNECTION` instead of uploading the credentials JSON file to BucketFS. This does not expose GCP credentials in BucketFS and it is easier to configure. See the [user guide](../user_guide/user_guide.md#configure-gcp-credentials) for details.
```

Comment:
```suggestion
This release allows configuring Google Cloud Storage (GCS) via a `CONNECTION` instead of uploading the credentials JSON file to BucketFS. This avoids exposing GCP credentials as file in BucketFS and simplifies configuration. See the [user guide](../user_guide/user_guide.md#configure-gcp-credentials) for details.
```